packages.config

```diff
@@ -6,4 +6,5 @@
   <package id="xunit.runner.console" version="2.0.0" />
   <package id="vswhere" version="1.0.62" />
   <package id="Microsoft.Net.Compilers" version="2.2.0" />
+  <package id="Octokit" version="0.26.0" />

```

Comment:
I feel a little bad making everyone pick this up, even non-owners, but it's probably not such a big deal.

Reply:
I'm sure nobody will mind ;)